cmake/OpenCVFindLibsPerf.cmake

```diff
@@ -41,11 +41,33 @@ endif(WITH_CUDA)
 
 # --- Eigen ---
 if(WITH_EIGEN)
-  find_path(EIGEN_INCLUDE_PATH "Eigen/Core"
-            PATHS /usr/local /opt /usr $ENV{EIGEN_ROOT}/include ENV ProgramFiles ENV ProgramW6432
-            PATH_SUFFIXES include/eigen3 include/eigen2 Eigen/include/eigen3 Eigen/include/eigen2
-            DOC "The path to Eigen3/Eigen2 headers"
-            CMAKE_FIND_ROOT_PATH_BOTH)
+  find_package (Eigen3)
```

Comment:
There is no such package in CMake 2.8.12 (Ubuntu 14.04) at least.
Need to guard this statement by check with proper CMake version and "OR DEFINED Eigen3_DIR".

Use "QUIET" option to avoid unnecessary messages.

Reply:
The package is not part of CMake but installed by Eigen itself.